cartographer_ros/CMakeLists.txt

```diff
@@ -48,6 +48,11 @@ find_package(cartographer REQUIRED)
 include("${CARTOGRAPHER_CMAKE_DIR}/functions.cmake")
 option(BUILD_GRPC "build features that require Cartographer gRPC support" false)
 google_initialize_cartographer_project()
+# PCL requires C++14 since version 1.10.0 - override if needed.
```

Comment:
Would it make sense to not explicitly set C++11 in `google_initialize_cartographer_project()`?

Reply:
cartographer-project/cartographer#1724

Reply:
Thanks, I will change it after that's merged.